language_service/src/signature_help.rs

```diff
@@ -36,6 +41,29 @@ pub(crate) fn get_signature_help(
 
     finder.visit_package(user_ast_package);
 
+    if position_encoding == Encoding::Utf16 {
+        // Fix up parameter offsets to be utf-16 based.
+        if let Some(ref mut sig_help) = finder.signature_help {
+            for sig in &mut sig_help.signatures {
+                for param in &mut sig.parameters {
+                    let range = Range::from_span(
+                        position_encoding,
+                        &sig.label,
+                        &Span {
+                            lo: param.label.0,
+                            hi: param.label.1,
+                        },
+                    );
+                    assert!(
```

Comment:
Were you running into problems with signature help having line-breaks?

Reply:
Not at all! I'm just being paranoid here - if we add line breaks to signature help down the line, this assert should remind us the offsets will be broken.